serpentTools/parsers/depletion.py

```diff
@@ -132,11 +135,19 @@ def _processChunk(self, chunk, name, variable):
         self.materials[name].addData(variable, cleaned)
 
     def _precheck(self):
-        """todo
+        """do a quick scan to see how many materials appear in the file
         """
-        pass
+        with open(self.filePath) as fh:
+            for line in fh:
+                sline = line.split()
+                if sline == []:
+                    continue
+                elif 'MAT' == line.split()[0]:
+                    self.materialCount += 1
```

Comment:
I think, since we may not store all materials, and the user may want to store the `TOT` values, we should just check to make sure this is in fact a `_dep.m` file. If `MAT` (or `DET` for detector) is not found in the file, the user should be notified.

Reply:
Sounds good. Done.